### ISSUE TITLE
Game doesn't start on my phone

### ISSUE THREAD

**Issue by tonybird** [nkomirisetti/MinuteWinners#2]
When I open the game on my phone (Safari, iOS 13.4) as instructed, I see a button labeled "Start!" but nothing happens when I press it. I tried restarting the browser and refreshing the page, but I still had the same issue. I was able to successfully start the game on my computer, but since it uses the accelerometer I wasn't really able to play.

**Comment by KevinLane36** [comment]
Hi Tony! 
Thanks for letting us know about this. We will attempt to replicate it on our own devices.
Do you know of you have cookies enabled with your device?

**Comment by tonybird** [comment]
Hey Kevin! Yes, I have cookies enabled. When I load the page right now I just see “Application error.”

**Comment by KevinLane36** [comment]
Can you try refreshing the page? I made a few changes and the site was down for a bit.

**Comment by nkomirisetti** [comment]
@tonybird did you make sure your ringer is on and that your volume was up?

**Comment by tonybird** [comment]
Yeah ringer is on and volume is up, I just sent Kevin a screen recording

**Comment by tsionc** [comment]
I am also not able to play the game its not starting in any of my devices

**Comment by jessti6** [comment]
it's also cannot play on my phone

**Comment by nkomirisetti** [comment]
Have you all made sure that the url said https and not http?  This URL (`https://minute-winners.herokuapp.com/`) should work hopefully.

**Comment by KevinLane36** [comment]
Could you please try to describe the behavior a bit when you go to the website?
Are you generally just stuck on the "Start" screen without being able to click past "Start"?

**Comment by tonybird** [comment]
This resolved the issue for me, thanks! You might want to consider redirecting users to HTTPS.
